src/components/Nfts/NftNetworkFilter.tsx

```diff
@@ -0,0 +1,150 @@
+import { CloseIcon } from '@chakra-ui/icons'
+import {
+  Button,
+  ButtonGroup,
+  Divider,
+  Flex,
+  IconButton,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  useColorModeValue,
+  useOutsideClick,
+} from '@chakra-ui/react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { useMemo, useRef, useState } from 'react'
+import type { FieldValues } from 'react-hook-form'
+import { useForm } from 'react-hook-form'
+import { IoOptionsOutline } from 'react-icons/io5'
+import { useTranslate } from 'react-polyglot'
+import { FilterGroup } from 'components/FilterGroup'
+import { Text } from 'components/Text'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { isSome } from 'lib/utils'
+
+export enum FilterFormFields {
+  Network = 'network',
+}
+
+type NftNetworkFilterProps = {
+  setFilters: Function
+  resetFilters: Function
+  hasAppliedFilter?: boolean
+  availableChainIds: ChainId[]
+}
+
+export const NftNetworkFilter = ({
+  setFilters,
+  resetFilters,
+  hasAppliedFilter = false,
+  availableChainIds,
+}: NftNetworkFilterProps) => {
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
May or may not be able to use a `useToggle` implementation but let's be safe, this is pretty much a stripped down, NFT network selection repurposed implementation of `TransactionHistoryFilter` - the goal is to get this working and merged, not cleanup the `<TransactionHistoryFilter />` (and this one while at it) or abstract things away